.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
```

Comment:
For these more util=y kinda ones, especially since they are the same between this PR and the other one, I would make them its own PR and merge that instead, so this PR is just focused on the datapipe bit, and not the extra formatting bits.